tools-local/tasks/Microsoft.DotNet.SourceBuild.Tasks.XPlat/DownloadFileSB.cs

```diff
@@ -0,0 +1,314 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Seeing this again with two files, I think it might be useful to move them to a new directory inside this project so they're more visible and easier to understand as a pair. (`CherryPickedTasks/` maybe?)

Reply:
Now that there's only one file (:+1:) a subdir isn't as useful, marking resolved.